Scripts/Utilities/MaterialUtils.cs

```diff
@@ -105,6 +105,14 @@ public static Color RandomColor()
 			return new Color(r, g, b);
 		}
 
+		public static Color HueShift(Color color, float shift)
```

Comment:
I tried that at first but it's more code. Here's what an extension method would look like:
>public static void HueShift(this Color color, float shift)
>{
>    Vector3 hsv;
>    Color.RGBToHSV(color, out hsv.x, out hsv.y, out hsv.z);
>    hsv.x = Mathf.Repeat(hsv.x + shift, 1f);
>    var c = Color.HSVToRGB(hsv.x, hsv.y, hsv.z);
>    color.r = c.r;
>    color.g = c.g;
>    color.b = c.b;
>    color.a = c.a;
>}